packages/gimbal/docs/ci/bitbucket/bitbucket-pipelines.yml

```diff
@@ -0,0 +1,11 @@
+image: moduscreate/gimbal:latest
```

Comment:
I'm wondering if we should show a more fleshed out version? The CircleCI version has things split out more: https://github.com/ModusCreateOrg/gimbal/blob/master/packages/gimbal/docs/ci/circleci/sample.yml

Also, if we do flesh out more, the `image` can be on the `step` for BB also (I'm doing that in client project atm).

Reply:
Could be nice. Yeah, it may be easier to realize how to attach gimbal steps in their current pipeline.

Reply:
@mitchellsimoens I have updated the example file, trying to set it to a minimum file in a way someone could easily integrate to their pipeline (even if they don't have a real CI flow). And, at the same time, I provide a gimbal only file example and a complete file considering a CI flow.

My intention is to ease adoption and first use. Someone that just have a simple pipeline can use gimbal just as someone that already has a full CI.

What do you think?